qcfractal/alembic/versions/cd3d3e2915ce_add_indices_to_access_and_base_result.py

```diff
@@ -0,0 +1,34 @@
+"""Add indices to access and base_result
+
+Revision ID: cd3d3e2915ce
+Revises: c05d63683601
+Create Date: 2021-03-08 09:40:56.987985
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "cd3d3e2915ce"
+down_revision = "c05d63683601"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_index("access_type", table_name="access_log")
+    op.create_index(op.f("ix_access_log_access_date"), "access_log", ["access_date"], unique=False)
+    op.create_index(op.f("ix_access_log_access_type"), "access_log", ["access_type"], unique=False)
```

Comment:
I think `unique=False` is the default, so no change on access_type index

Reply:
No change, but the name has changed (the previous index named `access_type` was actually an index on `access_date`. So this renames that, plus adds an index on date (probably needed for the dashboard). The unique=False comes from alembic autogeneration